components/Activity/ActivityEditor.tsx

```diff
@@ -271,18 +271,19 @@ export class ActivityEditor extends PureComponent<ActivityEditorProps> {
 
         <Form.Group as={Row} className="mb-3" controlId="briefInfo">
           <Form.Label column sm={2}>
-            {t('hackathon_detail')}
+            {t('activity_detail')}
             <span className="text-danger"> *</span>
           </Form.Label>
           <Col sm={10}>
             <HTMLEditor
-              name="detail"
               defaultValue={detail}
               onChange={code => (this.detailHTML = code)}
             />
             <Form.Control
+              name="detail"
               className="d-none"
```

Comment:
```suggestion
              hidden
              name="detail"
```
1. 越通用的属性越往前放
2. https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden